Mathlib.lean

```diff
@@ -876,6 +876,7 @@ import Mathlib.Algebra.Order.SuccPred
 import Mathlib.Algebra.Order.SuccPred.TypeTags
 import Mathlib.Algebra.Order.SuccPred.WithBot
 import Mathlib.Algebra.Order.Sum
+import Mathlib.Algebra.Order.ToBase
```

Comment:
```suggestion
import Mathlib.Algebra.Order.WithTop.ToBase
```
or something. Otherwise it's too generically named

Reply:
Implemented